src/helpers/titleToLong.js

```diff
@@ -0,0 +1,12 @@
+function titleToLong(title) {
+    if (title.length > 100)
+    {
+        return title.substring(0, 101) + "...";
+    }
+    else
+    {
+        return title;
+    }
```

Comment:
Als er niets meer na de else komt, dan kun je dit nog versimpelen:
```suggestion
    return title;
```